server/src/main.rs

```diff
@@ -161,8 +162,7 @@ fn main() {
         .options(Configuration::pinned().common.clone())
         .build()
         .expect("task_center builds");
-    tc.block_on({
-        let tc = tc.clone();
+    tc.handle().block_on({
```

Comment:
nit: Is `block_on` something that `OwnedTaskHandle` could expose directly? Then one does not have to go through the non-owned handle.

Reply:
Yes, this can be done indeed. Plans for `OwnedHandle` are not fleshed out yet tbh.